google-cloud-datastore/src/main/java/com/google/cloud/datastore/DatastoreException.java

```diff
@@ -38,13 +38,18 @@
   private static final long serialVersionUID = 2663750991205874435L;
```

Comment:
It is unclear to me (@eddavisson may add some clarity) if DEADLINE_EXCEEDED or UNAVAILABLE should be treated as "rejected" suggesting the server did not process the request (the latter sounds more like it than the former).
And if not "rejected" are those still safe to retry the commit (If not, doc should be clear about it).